src/repository.ts

```diff
@@ -81,7 +81,10 @@ abstract class Repository extends AsyncOptionalCreatable<RepositoryOptions> {
   }
 
   public getBranchName(): string {
-    const branch = this.env.getString('CIRCLE_BRANCH', exec('git branch --show-current', { silent: true }).stdout);
+    const branch = this.env.getString(
+      'CIRCLE_BRANCH',
+      shelljs.exec('git branch --show-current', { silent: true }).stdout
+    );
```

Comment:
this method can be deleted.

Reply:
cool, I think the one thing that was using it can go, too